soc/nordic/nrf54h/Kconfig

```diff
@@ -28,6 +28,7 @@ config SOC_NRF54H20_CPUAPP
 	select NRFS_HAS_VBUS_DETECTOR_SERVICE
 	select HAS_PM
 	select HAS_POWEROFF
+	select ARM_ON_ENTER_CPU_IDLE_HOOK if PM || POWEROFF
```

Comment:
Is this still required considering we changed the default to POWERON.ACT=1

Reply:
Yes - the hook is applied when DOMAIN_0 is not powered all the time - for proper idle execution. When no PM API is used, the domain is always on and no hook is required